scripts/docker/latest.deploy.sh

```diff
@@ -0,0 +1,3 @@
+docker build -t register-service:latest ./src/RegisterService
+docker tag register-service:latest brdv/register-service:latest
+docker push brdv/register-service:latest
```

Comment:
Waarom doe je een push? als je alles lokaal draait is het voldoende om lokaal te bouwen en te taggen toch?